### ISSUE TITLE
Straighterline-Heather Combs (Long Post)

### ISSUE THREAD

**Issue by jaahmuhl** [promazo/Content-Team#245]

**Comment by jaahmuhl** [comment]
**Raw Content:** https://drive.google.com/drive/folders/1TmXOGkRkyOuavD5qWG2qYO5bEkdAqoXy?usp=share_link

**Transcript:** https://www.rev.com/transcript-editor/shared/xTUUn4wDujydoIfDegFCBHA2ed0lh2Ohx9VMpSj5-8ViOXHOCqP-ywO8zgRlnoOO2hPCnaPK8jEw3o0j2ha3stUXPNM?loadFrom=SharedLink

**Comment by James-Moore18** [comment]
Clip: 48:51-49:23 
OFFICE HRS GENERAL INTRO:
Hello hello everyone (EXCITED AND WITH ENERGY) welcome back to Office Hours. A show about how to make a successful jump from college to young professional life. We know that landing your first job can often be an intimidating, complicated and stressful process, but its not impossible! (UP ENERGY) And we’re here to help by giving you access to some of the brightest minds in business to get their advice for how to build a successful career. Take what you can (short pause), pick what you like and listen for that stroke of brilliance that you can use to help you get hired. (edited)

BREAK

I’m James Moore and on the show today our host Brett is speaking with the CEO of Straighterline, Heather Combs. Heather is an extremely thoughtful leader, who is in constant pursuit of new knowledge and is always looking to learn, whether that be from a book or a mentee. She is a true go-getter, and provides exceptional advice throughout the conversation for everyone, in part due to her abundance of past experiences.
BREAK

A common piece of advice that most of people receive, is that they should learn how to say “No”. They warn you of the dangers of accepting too many opportunities, and the toll it can take to be stretched so thin you don’t have time for yourself. There is some merit to this advice, because there are only 24 hours in a day and taking care of your mental health should always be a priority. However, this advice ignores the potential doors that you could be closing by saying no too often. We all have had that moment where we have some great experience or meet someone that was the unexpected result of doing something we didn’t really want to do, yet the importance of learning to say no continues to be harped on. That’s why it was refreshing to hear todays guest, Heather Combs, speak about the vast benefits that can come from saying yes, and why young professionals especially should be saying yes quite often. Even if the opportunity doesn’t lead to some massive promotion or gaining a valuable connection, you likely learned something, which should never be discounted. 

Straighterline Info:
In 2008 Burck Smith founded Straighterline after questioning why the price of education was increasing and student outcomes were getting worse in spite of the advancements in technology. Today Straighterline has enrolled over 300,000 students in their courses and support over 45,000 a year with their affordable and accelerated learning pathways as an alternative to the traditional college model. And it's the perfect fit for those people who don’t have the luxury of putting their entire life on hold for 4 years, and it is exactly who Straighterline is looking to help.

Heather Info:

While in college Heather was working on graduating from Texas A&M with a degree double majoring in Political Science and Psychology, with a minor in philosophy she had her sights set on entering the Political sphere. After a brief stint working with a congressman from Texas she quickly realized that she didn’t actually want to be in the political world, though she did enjoy being in Washington DC. So after graduate school she returned to DC and landed a sales role after learning that she had a true talent for the profession due to her competitive nature and people skills. Now the CEO of Straighterline, Heather is focused on helping people to learn and then apply what they learned on a much quicker turnaround than the traditional learning  model. As she said in our discussion, in the modern world “who's got four years to wait on the next software developer to be ready to code something into existence or the next nurse to be ready to care for a patient when we have a shortage in those areas today.” Aside from a deep dive on the company, she also touches on effective conversation in a group setting, networking, the power of saying yes, why we need to keep learning, along with her perspective on mentor-mentee relationships and how she thinks either side can make the experience more beneficial for both parties....(Fade Out)

**Comment by jaahmuhl** [comment]
@James-Moore18 Needs a caption

**Comment by jaahmuhl** [comment]
@James-Moore18 really good intro

**Comment by jaahmuhl** [comment]
@James-Moore18 Need to say where the cuts for the podcast should go (where we want to start and where we want to end the podcast)

**Comment by James-Moore18** [comment]
In 2008 a man by the name of Burck Smith was frustrated with what education had evolved into and questioned why both the price and the student outcomes were getting worse in spite of technological advancements. He knew there had to be a better way and he founded Straighterline to help get students to the workforce faster so they can apply what they had learned quickly, reducing the cost of education while improving outcomes. That theory proved to be spot on and today Straighterline is led by our guest, CEO Heather Combs, with the platform supporting over 45,000 students per year. Heather is a big believer in the power of saying yes to new opportunities and advocates for constantly trying to learn new skills and information, especially after graduation. In this interview Heather talks about how she got to be CEO of Straighterline, tries to dispel some of the negative connotations surrounding jobs in sales, and gives some great advice on how to take your mentor-mentee relationship to the next level.

**Comment by jaahmuhl** [comment]
@James-Moore18 Need podcast starting point and end point as well as if there are any glitches in the audio that need to be edited out.

**Comment by James-Moore18** [comment]
Podcast:
4:08 - 58:03

No audio glitches

**Comment by jaahmuhl** [comment]
@James-Moore18 ready for recording

**Comment by jaahmuhl** [comment]
@James-Moore18 ready to record

**Comment by jaahmuhl** [comment]
Podcast Intros: https://drive.google.com/drive/folders/1h43BQmJ1unvGZDenUBZpwjmxz_PpyxgR?usp=drive_link

**Comment by jaahmuhl** [comment]
Podcast Intros: https://drive.google.com/drive/folders/1BgcJmltt_2-nXq_Ge_eYZQxl8uES85O-?usp=drive_link
